package.json

```diff
@@ -3,7 +3,7 @@
     "command": "codegen",
     "version": "0.0.40",
     "description": "Command for generating code from kapeta YML files",
-    "main": "index.js",
+    "main": "dist/index.js",
```

Comment:
Is it `dist/index.js` or `dist/src/index`? 👀 Codegen above says `src` too

Reply:
I have a dist/index.js which is the src/index.ts file so I guess it should be fine